components/checkbox/skin.css

```diff
@@ -10,106 +10,96 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 }
 
 .spectrum-Checkbox:hover {
   &.is-indeterminate .spectrum-Checkbox-box,
   .spectrum-Checkbox-input:checked + .spectrum-Checkbox-box {
-    border-color: var(--spectrum-checkbox-emphasized-box-border-color-selected-hover);
-
+    border-color: var(--spectrum-checkbox-box-border-color-selected-hover);
   }
 }
 
 .spectrum-Checkbox:active {
   &.is-indeterminate .spectrum-Checkbox-box,
   .spectrum-Checkbox-input:checked + .spectrum-Checkbox-box {
-    border-color: var(--spectrum-checkbox-emphasized-box-border-color-selected-down);
+    border-color: var(--spectrum-checkbox-box-border-color-selected-down);
 
   }
 }
 
 .spectrum-Checkbox {
-  border-color: var(--spectrum-checkbox-emphasized-box-border-color);
+  border-color: var(--spectrum-checkbox-box-border-color);
 
   &:hover {
     .spectrum-Checkbox-box {
-      border-color: var(--spectrum-checkbox-emphasized-box-border-color-hover);
-
-      box-shadow: none;
+      border-color: var(--spectrum-checkbox-box-border-color-hover);
     }
     .spectrum-Checkbox-label {
-      color: var(--spectrum-checkbox-emphasized-text-color-hover);
+      color: var(--spectrum-checkbox-text-color-hover);
     }
   }
 
   &:active {
     .spectrum-Checkbox-box {
-      border-color: var(--spectrum-checkbox-emphasized-box-border-color-down);
+      border-color: var(--spectrum-checkbox-box-border-color-down);
 
     }
     .spectrum-Checkbox-label {
-      color: var(--spectrum-checkbox-emphasized-text-color-down);
+      color: var(--spectrum-checkbox-text-color-down);
     }
   }
-
-  .spectrum-Checkbox-checkmark,
-  .spectrum-Checkbox-partialCheckmark {
-    color: var(--spectrum-checkbox-emphasized-checkmark-color);
-  }
 }
 
-.spectrum-Checkbox-input {
+.spectrum-Checkbox .spectrum-Checkbox-input,
+.spectrum-Checkbox .spectrum-Checkbox-input:checked {
   &:disabled + .spectrum-Checkbox-box {
-    /* Use important to override hover states */
-    border-color: var(--spectrum-checkbox-emphasized-box-border-color-disabled) !important;
-
-    background-color: var(--spectrum-checkbox-emphasized-box-background-color-disabled);
+    border-color: var(--spectrum-checkbox-box-border-color-disabled);
+    background-color: var(--spectrum-checkbox-box-background-color-disabled);
   }
 
   &:disabled ~ .spectrum-Checkbox-label {
-    color: var(--spectrum-checkbox-emphasized-text-color-disabled);
+    color: var(--spectrum-checkbox-text-color-disabled);
   }
 }
 
 /* Focus */
 .spectrum-Checkbox-input {
   &:focus-ring + .spectrum-Checkbox-box {
-    /* Since the specificity of .focus-ring is less than :hover, we need important */
-    border-color: var(--spectrum-checkbox-emphasized-box-border-color-key-focus) !important;
-
-    box-shadow: 0 0 0 1px var(--spectrum-checkbox-emphasized-box-border-color-key-focus) !important;
+    box-shadow:  0 0 0 2px var(--spectrum-checkbox-focus-ring-gap-color-key-focus), 0 0 0 4px var(--spectrum-checkbox-focus-ring-color-key-focus);
```

Comment:
Ideally, you would create a global DNA var (by adding a [static alias](https://git.corp.adobe.com/Spectrum/spectrum-dna/blob/master/data/aliases/StaticAliases.mjs)) for the focus ring width and offset, and use that (if it doesn't already exist)...

If you don't do that, note that you can use any global DNA vars (non-scale dependent) inside of `skin.css`:

* `--spectrum-alias-border-size-thick` or `--spectrum-global-dimension-static-size-25` for 2px
* `--spectrum-alias-border-size-thicker` or `--spectrum-global-dimension-static-size-50` for 4px

As a last resort, you can put them in `commons/index.css` or create a `commons/skin.css` and import it inside of `commons/index.css`, but it would be best to create a DNA var. Can you create static aliases instead?